MapboxCoreNavigation/RouteController.swift

```diff
@@ -185,6 +185,10 @@ open class RouteController: NSObject {
         self.resumeNotifications()
         self.startEvents(route: route)
         self.resetSession()
+        
+        if !(route.routeOptions is NavigationRouteOptions) {
+            print("`Route` was created using `RouteOptions` and not `NavigationRouteOptions`. Although not required, this may lead to a suboptimal navigation experience. Without `NavigationRouteOptions`, it is not guaranteed you will get congestion along the route line, better ETAs and ETA label color dependent on congestion.")
```

Comment:
None of these features are implemented in Core Navigation, only in MapboxNavigation. Can we move this check to NavigationViewController’s initializer?

Also, use a (lazily initialized) global or static property to track whether this message has been printed before and avoid printing it a second time.